### PR TITLE
Incorporation of Material Design Lite #1

### PR DESCRIPTION
**Pull request by ghost**
- applied Material Design Lite styling, particularly in the lists
- replaced <table> with cards in a grid
- elaborated descriptions of the usage of each resource link

### CODE REVIEW

File: resources.html [hunk 1]
```diff
@@ -22,83 +34,213 @@
...
+        <div class = "mdl-card__supporting-text">
+          <ul class = "mdl-list">
+            <li class = "mdl-list__item">
+              <span class = "mdl-list__item-text-body"><h3>
+                (\ (\<br>
+                ( . .)<br>
+                C(")(")</h3>
+              </span>
+            </li>
+            <li class = "mdl-list__item">I do not know what to do with this space.</li>
```
Comment (WilliamC07) [inline resources.html]: remove

**Review by WilliamC07 — CHANGES_REQUESTED** [review]
When listing websites, can we format it like this: https://ibb.co/NW925xM (this is a random website i found for uploading images)
I would use the "Two line" design: https://getmdl.io/components/index.html#lists-section
Right now, the links and description do not align.

I also made one comment to line 237.

Everything else looks great. Thank you.

**Comment by ghost** [comment]
- applied three line design to the lists
- resolved comments

**Review by WilliamC07 — APPROVED** [review]

**Comment by WilliamC07** [comment]
Looks really nice thanks.

**Pull request merged**